cli/cli_app.go

```diff
@@ -4,7 +4,7 @@ import (
 	"encoding/json"
 	"fmt"
 	"io"
-	"log"
+	log "github.com/sirupsen/logrus"
```

Comment:
NIT: any reason to `import` this under the name `log` instead of `logrus`? It took me a while to figure out that `log` was really `logrus`, so perhaps it'll be confusing for others too?

Reply:
Importing `logrus` as `log` allows accessing default logging object. Ans since `logrus` can be used as a drop-in-replacement for `log`, it generally causes no issues. But I will change this to usual imports to avoid confusion.

Reply:
Roger!